addons/base_import/tests/test_base_import.py

```diff
@@ -707,6 +707,57 @@ def test_skip_record_import(self):
         # if results empty, no errors
         self.assertItemsEqual(results['messages'], [])
 
+    def test_multi_mapping(self):
```

Comment:
Ok for the test, and no fix related to it ? "This commit is done as a preliminary step to then improve the performance of
the method that handles the multi-mapping." -> is it planned for this PR ? Of master only ? missing some context here :)